.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,33 @@
-## Testing
+## ℹ️ Description
 
 <!--
-What is needed to test the changes? e.g. specific cluster, service definitions
-How can one see the result of your work? e.g. configurations, URLs
+What is the change your PR does?
+Please add a short description here. (one sentence is enough)
 -->
 
-## Trade-offs
+- 🔖 JIRA: [DCOS-12345](https://jira.mesosphere.com/browse/DCOS-12345)
+  <!-- add things that reviewers need to know, e.g.
+- 🚫 This blocks #PRNumber
```

Comment:
Why is this important to know for a reviewer?

Reply:
it might be important because the blocked PR blocks a release and this enables it. 
Also, this is an example, what is important in a particular PR decides the creator 🤷‍♂️

Reply:
But if it's important that should be reflected in Jira and or it should be flagged here via a label so it draws more attention. I would assume that most people don't read every github pr description before they decide which to review

Reply:
are we discussing the example here?! I'm happy to remove them, but still there can be situations, where the PR creator might want to _help_ reviewers.

Reply:
Can you explain what that might be? The only things that pop in my head can be sorted into either the review section or the trade-offs section, so I am a bit puzzled what would go here.

Reply:
as stated below, in some situations it might be necessary or important for the creator to highlight certain things at the very beginning. labels are something else tho, because they declare a state and do not transport information (as much as text can)

Reply:
> important for the creator to highlight certain things at the very beginning